### PR TITLE
Changed error codes for examples

### PR DESCRIPTION
**Pull request by kgorecki**
Resolves issue #247

### CODE REVIEW

File: examples/bitmap_operation/example_bitmap_operation.cpp [hunk 1]
```diff
@@ -24,17 +24,18 @@ int main()
         // Second way to do
         method2();
     }
-    catch( const imageException & ex ) {
-        // uh-oh, something went wrong!
+    catch( imageException & ex ) { // uh-oh, something went wrong!
```
Comment (ihhub) [inline examples/bitmap_operation/example_bitmap_operation.cpp]: Please remove this catch block from each file. As it's mentioned in the issue we replace `imageException` by `std::exception`  in catch block. `imageException` class is inherited from `std::exception` base class so catching base class exception would catching inherited (class) exception too.

File: examples/bitmap_operation/example_bitmap_operation.cpp [hunk 2]
```diff
@@ -24,17 +24,14 @@ int main()
...
```
Comment (ihhub) [inline examples/bitmap_operation/example_bitmap_operation.cpp]: Please follow the structure which I wrote in the original issue:
```cpp
catch( const std::exception & ex ) { // uh-oh, something went wrong!
    std::cout << "Exception " << ex.what() << " raised. Closing the application..." << std::endl;
    return 1;
}
catch( ... ) { // uh-oh, something terrible happen!
    std::cout << "Generic exception raised. Closing the application..." << std::endl;
    return 2;
}

std::cout << "Application ended correctly." << std::endl;
return 0;
```

Just copy-paste the code in all files :)
Only for below files we don't need `std::cout << "Application ended correctly." << std::endl;`:
```bash
examples/cuda/pinned_memory/pinned_memory.cpp
examples/qt_framework/qimage_display/main.cpp
examples/raspicam/main.cpp
examples/thread_pool/example_thread_pool.cpp
```
Above files output or show something so we don't need to output last message which is an indicator that application completed it's work.

Comment (kgorecki) [reply examples/bitmap_operation/example_bitmap_operation.cpp]: Sorry for that, I was in hurry with last commit and totally messed stuff.
I also removed unreachable "return 0;" from qt_framework/qimage_display/main.cpp

Comment (ihhub) [reply examples/bitmap_operation/example_bitmap_operation.cpp]: No blame on you :) We all make mistakes ;)

Comment (kgorecki) [reply examples/bitmap_operation/example_bitmap_operation.cpp]: True, but I should have followed the rule: first think, than do, than commit ;}

Comment (ihhub) [reply examples/bitmap_operation/example_bitmap_operation.cpp]: I fixed that,

**Comment by kgorecki** [comment]
@ihhub "return 0;" line in Qt example is unreachable now, so I'd suggest to take it away or change it to "return app.exec();" (current line 84).

**Comment by ihhub** [comment]
@kgorecki yes, you're totally write. I revert it back.

**Comment by ihhub** [comment]
@kgorecki thanks a lot for continuous support of th project!

**Pull request merged**